ios/brave-ios/Sources/Brave/Assets/Images.xcassets/NTT/ntt_cancel_button.imageset/Contents.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Is there a reason we cannot add as SVGs?

Reply:
Can we also add _video to the name such as `ntt_video_cancel_button`, same for other assets

Reply:
Will be replaced by Leo SF Symbol icons